src/idea.js

```diff
@@ -13,7 +13,15 @@ class Idea {
    }
 
   deleteFromStorage() {
-   localStorage.removeItem(this.id);
+    getStoredIdeas()
```

Comment:
It looks like we are running a lot of repeat functionality within our Idea class and the function that invokes this method. Let's take a closer look at this with the next branch